docs/editor_manual/editing_existing_pages.md

```diff
@@ -10,15 +10,16 @@ Here is how you can access the edit screen of an existing page:
 
 -   When editing an existing page the title of the page being edited is displayed at the top of the page.
 -   You can perform various actions (such as copy, move or delete a page) by opening the secondary menu at the right of the top breadcrumb.
--   Underneath the title, you can find the page type and a link to the page history.
+-   The header has a link to the page history.
+-   In the Info side panel, you can find the page type and other important page information.
 -   If published, a link to the live version of the page is shown on the top right.
 -   You can toggle commenting mode by clicking the icon in the top right, which also shows the number of comments.
 -   You can change the title of the page by clicking into the title field.
 -   When you're typing into a field, help text is often displayed on the right-hand side of the screen.
```

Comment:
The writing is a bit inconsistent here, and could introduce side panels better, but I chose to keep it here for now.